io.asgardio.dotnet.oidc.sdk.sample.setup/OAuthConfigDialog.wxs

```diff
@@ -0,0 +1,62 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<!--
+  ~ Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~ WSO2 Inc. licenses this file to you under the Apache License,
+  ~ Version 2.0 (the "License"); you may not use this file except
+  ~ in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+-->
+
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
+  <Fragment>
+    <UI>
+      <Dialog Id="OAuthConfigDialog" Width="370" Height="427" Title="[ProductName] [Setup]" NoMinimize="yes">
+        <Control Id="ClientIdLabel" Type="Text" X="45" Y="73" Width="100" Height="15" TabSkip="no" Text="&amp;Client ID:" />
+        <Control Id="ClientIdEdit" Type="Edit" X="45" Y="85" Width="265" Height="18" Property="CLIENT_ID" Text="{80}" />
+        <Control Id="ClientSecretLabel" Type="Text" X="45" Y="110" Width="100" Height="15" TabSkip="no" Text="&amp;Client Secret:" />
+        <Control Id="ClientSecretEdit" Type="Edit" X="45" Y="122" Width="265" Height="18" Property="CLIENT_SECRET" Text="{80}" />
+        <Control Id="AuthorizationEndpointLabel" Type="Text" X="45" Y="147" Width="100" Height="15" TabSkip="no" Text="&amp;Authorization Endpoint:" />
+        <Control Id="AuthorizationEndpointEdit" Type="Edit" X="45" Y="159" Width="265" Height="18" Property="AUTHORIZATION_ENDPOINT" Text="{80}" />
+        <Control Id="TokenEndpointLabel" Type="Text" X="45" Y="184" Width="100" Height="15" TabSkip="no" Text="&amp;Token Endpoint:" />
```

Comment:
Don't we have a way to reuse these styling?

Reply:
no, we have to specify them explicitly

Reply:
ack